server/package.json

```diff
@@ -62,10 +65,12 @@
     "bytes": "^3.1.2",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
+    "dataloader": "^2.2.2",
     "date-fns": "^2.30.0",
     "file-type": "16.5.4",
-    "graphql": "^16.8.0",
```

Comment:
Note: I had some issues with later versions and the conditionalSchema in yoga that we created, not quite sure what's the issue there yet but we need to find a solution

Reply:
@Weiko My PR on graphql-yoga has been stamp today, I still need to do another step, but it will get merged soon hopefully !